src/presentation/controllers/action-controller.ts

```diff
@@ -13,7 +13,7 @@ export class ActionController extends BaseController {
     super()
   }
 
-  @GetRequest()
+  @GetRequest('/rundowns/:rundownId')
```

Comment:
I know you made this change in preperation, but wouldn't it be better to make this change, when needed?

At the moment it will just result in us needing to send a rundownId, which will not be used, when performing the request.

Reply:
I think it is fine to have it setup already now, since it allows the frontend to implement logic towards it. And it will be needed when implementing some of the actions, e.g. video clips.